### PR TITLE
Service layer

### PR DESCRIPTION
**Pull request by edwardGr**
Implemented service layer. Added new property to UserEntity called contacts.

### CODE REVIEW

File: src/main/java/com/george/messenger/domain/dao/repository/MessageEntityRepository.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+import com.george.messenger.domain.dao.entity.MessageEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface MessageEntityRepository extends JpaRepository<MessageEntity, Long> {
+
+    List<MessageEntity> findByContentLike(String query);
+}
```
Comment (VadymPolyanski) [inline src/main/java/com/george/messenger/domain/dao/repository/MessageEntityRepository.java]: not String query
use org.springframework.data.jpa.repository.Query annotation

File: src/main/java/com/george/messenger/domain/services/GenericService.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package com.george.messenger.domain.services;
+
+import java.io.Serializable;
+import java.util.List;
+
+public interface GenericService<T, ID extends Serializable> {
+
+    List<T> save(Iterable<T> entities);
+
+    T save(T entity);
```
Comment (VadymPolyanski) [inline src/main/java/com/george/messenger/domain/services/GenericService.java]: well done

File: src/main/java/com/george/messenger/domain/dao/repository/MessageEntityRepository.java (hunk 1 above)
Comment (edwardGr) [inline src/main/java/com/george/messenger/domain/dao/repository/MessageEntityRepository.java]: Like this: 
`@Query("select m from MessageEntity m where m.content like %?1") `
`List<MessageEntity> findByContentLike(String query);`
?

Comment (VadymPolyanski) [inline src/main/java/com/george/messenger/domain/dao/repository/MessageEntityRepository.java]: @Query("select m from MessageEntity m where m.content like %?1")
List<MessageEntity> findByContentLike(String content);

**Comment by VadymPolyanski** [comment]
Edward, are you going to do something?

**Comment by edwardGr** [comment]
Definitely. Could you check skype?

**Pull request merged**